src/containers/BooksForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '../actions/index';
+
+const StyledBookForm = styled.form`
+  background: yellow;
+  color: green;
+
+  button {
+    color: blue;
+    background: red;
+  }
+`;
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+      title: '',
+      category: ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'],
+
+    };
+    this.count = 0;
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const { dispatch } = this.props;
+    const { addBook } = actions;
+    const input = e.target.querySelector('input').value;
+    const select = e.target.querySelector('select').value;
+    const newBook = {
+      title: input,
+      category: select,
+    };
+    dispatch(addBook(newBook));
+    this.setState({
+      title: '',
+    });
+  }
```

Comment:
[OPTIONAL] - **marking this as optional since it was missed in the previous review but I highly recommend you to work on this before merging**  

According to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896828/), you should use `mapDispatchToProps`, so please change this so that you can the `mapDispatchToProps` to connect to the store.
![Screenshot](https://user-images.githubusercontent.com/16196289/90236953-25a94900-de2c-11ea-8514-d6fd3b33c0c3.png)

- Apart from resetting the title after clicking submit, make sure you reset the category to `Action`